conda-recipe/meta.yaml

```diff
@@ -1,36 +1,43 @@
-{% set data = load_setup_py_data() %}
+{% set package_name = "pcdsutils" %}
+{% set import_name = "pcdsutils" %}
+{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
 
 package:
-    name    : pcdsutils
-    version : {{ data.get('version') }}
+  name: {{ package_name }}
+  version: {{ version }}
 
 source:
-    path: ..
+  path: ..
 
 build:
-    number: 1
-    noarch: python
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-    build:
-      - python >=3.7
-      - setuptools
-    run:
-      - python >=3.7
-      - prettytable
-      - pypandoc
-      - pyyaml
-      - qtpyinheritance
-      - requests
+  build:
+  - python >=3.7
```

Comment:
Do we still use anything below py3.9 here?  CI runs for 3.9+ anyway

Reply:
We don't run or test <3.9